src/sdam/topology.ts

```diff
@@ -496,14 +496,14 @@ export class Topology extends TypedEventEmitter<TopologyEvents> {
     if (typeof options === 'boolean') {
       options = { force: options };
     }
-    options = options ?? {};
+    options = options ?? { force: false };
 
     if (this.s.state === STATE_CLOSED || this.s.state === STATE_CLOSING) {
       return callback?.();
     }
 
     const destroyedServers = Array.from(this.s.servers.values(), server => {
-      return promisify(destroyServer)(server, this, options as CloseOptions);
+      return promisify(destroyServer)(server, this, Object.assign({ force: false }, options));
```

Comment:
We can again remove the override of the reordering of arguments, we only call this from the mongoclient with either options and no callback or options and callback, so the argument fixing to support callback with no options can go away, that will make the typescript happy on this line if you write:

```ts
return promisify(destroyServer)(server, this, { force: !!options?.force });
```